etc/ProdOfflineConfiguration.py

```diff
@@ -973,7 +973,7 @@
 for dataset in DATASETS:
     addDataset(tier0Config, dataset,
                do_reco=True,
-               raw_to_disk=True,
+               raw_to_disk=False,
```

Comment:
I think the purpose of this PR is the opposite. We want to send HLTPhysics and ZeroBias RAW to disk, while removing  Raw to disk subscriptions for every other dataset.
https://cms-talk.web.cern.ch/t/raw-on-disk-in-run2022b-and-later/12317